### PR TITLE
Flaresolverr

### PR DESCRIPTION
**Pull request by Findarato**
[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

------------------------------

 - [X] I have read the [contributing](https://github.com/linuxserver/reverse-proxy-confs/blob/master/.github/CONTRIBUTING.md) guideline and understand that I have made the correct modifications

------------------------------

## Description
I am adding support for the flaresolvrr container

## How Has This Been Tested?
I am running this config on my network currently and it works

### CODE REVIEW

File: flaresolverr.subfolder.conf.sample [hunk 1]
```diff
@@ -0,0 +1,26 @@
+## Version 2023/12/15
+# make sure that your flaresolverr container is named flaresolverr
+# make sure that sonarr is set to work with the base url /sonarr/
```
Comment (drizuid) [inline flaresolverr.subfolder.conf.sample:3]: this seems wrong...

**Review by drizuid — CHANGES_REQUESTED** [review]

**Comment by drizuid** [comment]
> I am running this config on my network currently and it works

you tested both subdomain and subfolder?

**Comment by STaRDoGG** [comment]
Works for me (subdomain)

File: flaresolverr.subfolder.conf.sample (hunk 1 above)
Comment (Findarato) [reply flaresolverr.subfolder.conf.sample:3]: Woops I will fix, copy pasta caught me

**Comment by drizuid** [comment]
once someone can test/confirm subfolder, i will merge. unfortunately, OP didn't answer the question and stardogg only tested subdomain.

**Review by drizuid — APPROVED** [review]

**Pull request merged**